modules/fedimint-ln/src/lib.rs

```diff
@@ -208,17 +209,46 @@ pub struct LightningGateway {
 }
 
 #[derive(Debug, Clone, PartialEq, Eq, Hash, Encodable, Decodable, Serialize, Deserialize)]
-pub struct LightningConsensusItem {
+pub struct DecryptionShareCI {
     pub contract_id: ContractId,
     pub share: PreimageDecryptionShare,
 }
 
 impl std::fmt::Display for LightningConsensusItem {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        write!(f, "LN Decryption Share for contract {}", self.contract_id)
+        match self {
+            LightningConsensusItem::RoundConsensus(rc) => {
+                write!(f, "Clock time {:?}", rc.clock_time)
+            }
+            LightningConsensusItem::DecryptionShare(decryption_share) => {
+                write!(
+                    f,
+                    "LN Decryption Share for contract {}",
+                    decryption_share.contract_id
+                )
+            }
+        }
     }
 }
 
+#[derive(
+    Debug, Clone, PartialEq, Eq, Hash, Encodable, Decodable, Serialize, Deserialize, UnzipConsensus,
+)]
+pub enum LightningConsensusItem {
+    RoundConsensus(RoundConsensusItem),
+    DecryptionShare(DecryptionShareCI),
+}
+
+#[derive(Clone, Debug, PartialEq, Eq, Hash, Serialize, Deserialize, Encodable, Decodable)]
+pub struct RoundConsensusItem {
+    pub clock_time: SystemTime,
```

Comment:
On reflection we might want to make this a `u64` or some other internal timestamp type to avoid platform-dependent behavior of `SystemTime` (maybe that's not the case, but should check, different OSs treat time differently)

Reply:
I converted clock_time to `u64` in https://github.com/fedimint/fedimint/pull/1074/commits/39304eadf2560d4f3d161d333f1d5a9666bafd0f